responsible-ai-collaborative/aiid
# Fix blog post opengraph image

Closes #1220

### Steps to test

1. Open preview link https://deploy-preview-1250--staging-aiid.netlify.app
2. Open any blog post
3. Copy the blog post URL. ie: https://deploy-preview-1250--staging-aiid.netlify.app/blog/multilingual-incident-reporting
4. Share the blog post link on
4.1 Twitter
4.2 LinkedIn
4.3 Facebook
5. Check if the image preview is displayed correctly on each post/tweet

### Twitter:

<img width="613" alt="image" src="https://user-images.githubusercontent.com/6564809/195424336-00133923-1e1b-493a-99cf-3a6df442dc7e.png">

### LinkedIn post:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/6564809/195424115-18d6e89f-6877-41f8-bffd-77a6edb338f8.png">
 
### Facebook post

<img width="697" alt="image" src="https://user-images.githubusercontent.com/6564809/195424589-d7ebc8b6-3792-4c1c-9cc1-a618b27456fd.png">
